### PR TITLE
upload_Derek BCO

### PR DESCRIPTION
**Pull request by cderek7**
read for review

### CODE REVIEW

File: Student_BCOs/Derek_Cao_BCO 000534.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "10/16/2019"
+            }
+        ],
```
Comment (jpat1546) [inline Student_BCOs/Derek_Cao_BCO 000534.json:26]: The previous pull request for this BCO had a usability domain that was a copy/paste from the abstract.  
I addressed this in my [review](https://github.com/biocompute-objects/GW-SMHS-BIOC6223/pull/194/files/13ee6ba52702179503df76d0d5a8c51c35e49248#r336080331) of that pull request.

----
Below I've copied your usability domain sentences and the publication text. 

- Usabiility domain:  Identify a link between STAT3 and E2F target genes and also alter the STAT family members depends on the STAT3 deletion
- publication text: `we identify a link between STAT3 and E2F target genes as well as alteration in STAT family members upon STAT3 deletion`

- Usabiility domain:  Identify multi-level (Bru-Seq, RNA-Seq, and MS Proteomic) expression signatures of parental and STAT3 KO ovarian cancer cells by using multi-omic genome-wide profiling.
- publication text:`We used multi-omic genome-wide profiling to identify multi-level (Bru-Seq, RNA-Seq, and MS Proteomic) expression signatures of STAT3 KO ovarian cancer cells.`

- Usabiility domain:  Identify new genes set and 41 genes differentially regulated in order to response for STAT3 KO within a wide range of cellular processes.
- publication text: `identified new gene sets and 41 genes differentially regulated in response to STAT3 KO involved in a wide range of cellular processes.`
----

### If this BCO is going to have an "approved" status, it needs an entirely reworded Usability domain.

@kee007ney

Comment (kee007ney) [reply Student_BCOs/Derek_Cao_BCO 000534.json:26]: @cderek7 It's really important that these have fully reworded descriptions stated in their entirety by you! Please write a few sentences of your own to describe the pipeline.

Comment (cderek7) [reply Student_BCOs/Derek_Cao_BCO 000534.json]: Yes! I reword all the sentences already!

**Comment by jpat1546** [comment]
Closed by #250

**Pull request closed without merging**